packages/lwt-windows.2.5.2/opam

```diff
@@ -0,0 +1,61 @@
+opam-version: "1.2"
+maintainer: "Pierre Boutillier <pierre.boutillier@laposte.net>"
+authors: [
+  "Jérôme Vouillon"
+  "Jérémie Dimino"
+]
+homepage: "http://github.com/ocsigen/lwt"
+dev-repo: "git://github.com/ocsigen/lwt"
+bug-reports: "http://github.com/ocsigen/lwt/issues"
+license: "LGPL with OpenSSL linking exception"
+
+build: [
+  ["ocaml" "setup.ml" "-configure"
+    "--override" "os_type" "Win32"
+    "--override" "flexlink" "%{lib}%/flexdll-windows/flexlink"
+    "--override" "ext_dll" ".dll"
+    "--override" "datarootdir" "%{prefix}%/windows-sysroot/share"
+    "--disable-pthread"
+    "--prefix" "%{prefix}%/windows-sysroot"
+    "--disable-libev"
+#    "--%{camlp4:enable}%-camlp4"
+    "--disable-camlp4"
+    "--%{react-windows:enable}%-react"
+    "--%{ssl-windows:enable}%-ssl"
+    "--%{base-unix:enable}%-unix"
+    "--%{base-threads:enable}%-preemptive"
+    "--%{lablgtk-windows:enable}%-glib"
+    "--disable-ppx"]
+  ["ocamlbuild" "-use-ocamlfind" "-just-plugin"]
+  ["env" "OCAMLFIND_TOOLCHAIN=windows" "ocaml" "setup.ml" "-build"]
+]
+install: [
+  ["env" "OCAMLFIND_TOOLCHAIN=windows" "ocaml" "setup.ml" "-install"]
+  ["ln" "-s" "../../bin/ppx_lwt" "%{prefix}%/windows-sysroot/bin/ppx_lwt"]
+
+  # tell ocamlfind that package lwt.ppx is 'available'
+  ["ln" "-s" "../../../lib/lwt/ppx.cma"
```

Comment:
Ok, this seems really gross. Can you please use sed to remove the `enable_if ...` line? It should be something like:
```
["sed" "-e" "/enable_if .../d" "-i" "%{prefix}%/windows-sysroot/lib/lwt/META"]
```